cinema/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
```

Comment:
Add one blank line between different groups of imports here and in the other places to follow PEP-8